packages/test-utils/src/utils.ts

```diff
@@ -12,7 +12,8 @@ import { axeRuntimeExceptionMsgPrefix } from '@sa11y/common';
  * Ref: https://eddmann.com/posts/cartesian-product-in-javascript/
  */
 // TODO(types): Fix types for the cartesianProduct function - from any to generics
-/* eslint-disable @typescript-eslint/no-explicit-any */
+// eslint-disable-next-line eslint-comments/disable-enable-pair
+/* eslint-disable @typescript-eslint/no-explicit-any, @typescript-eslint/no-unsafe-argument */
```

Comment:
nit: Can the `@typescript-eslint/no-unsafe-argument` be added to `eslint-enable` at end of the function to prevent having double line eslint disables?

Reply:
Fixed it by fixing up commit 7fdb4d5924ad806d2507514a3321eab21f405c43 @mohanraj-r